### PR TITLE
revise children of 'myopathy caused by variation in FKRP'

### LINKED ISSUE #6851: revise children of 'myopathy caused by variation in FKRP'
there are children that are incorrectly classified under this term, as pointed out by Shruthi at ClinGen.

cc @twhetzel

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
- exclude superclasses and add new super class to muscle-eye-brain disease
- exclude superclasses from 'muscular dystrophy-dystroglycanopathy, type A'

close #6851

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@twhetzel this is ready for review, thanks!

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -351050,7 +351054,7 @@ synonym: "MEB disease with bilateral multicystic leucodystrophy" EXACT [Orphanet
 xref: ICD10CM:G71.2 {source="Orphanet:370997/attributed", source="Orphanet:370997/ntbt", source="Orphanet:370997"}
 xref: Orphanet:370997 {source="MONDO:equivalentTo"}
 is_a: MONDO:0018276 ! muscular dystrophy-dystroglycanopathy
-is_a: MONDO:0018869 ! cobblestone lissencephaly
+is_a: MONDO:0018869 {source="Orphanet:370997"} ! cobblestone lissencephaly
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:351057]: @nicolevasilevsky can you help me understand why the `is_a` has a source, but in other cases when this class is used in an `excluded_subClassOf` relationship the source is not present?

**Review by twhetzel — APPROVED** [review]
Looks good. One question inline for my education :)

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:351057]: I don't know where all these unsupported annotations came from, we have a ticket for this: https://github.com/monarch-initiative/mondo/issues/5857

We initially thought we could just remove them all but as we started reviewing them, we realized some were legitimate.

**Pull request merged**